config/initializers/will_paginate_extensions.rb

```diff
@@ -0,0 +1 @@
+require 'will_paginate/array'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Final newline missing.